device-code-flow-console/MyInformation.cs

```diff
@@ -32,185 +31,46 @@
 
 namespace device_code_flow_console
 {
-    public class MyInformation
+    public class MyInformation : PublicAppUsingDeviceCodeFlow
```

Comment:
This inheritance is not natural. I would use composition instead. Makes for much more readable code.

Reply:
Done. Thanks!